uefi-test-runner/Cargo.toml

```diff
@@ -21,6 +20,9 @@ multi_processor = []
 # Enable the PXE test.
 pxe = []
 
+# Enable the `unstable` feature of the `uefi` crate.
+unstable = ["uefi/unstable"]
```

Comment:
feature name is `unstable`, command line argument is `--include-unstable` and enum variant is `TestUnstable`. Can we unify this at least a little?

Reply:
Updated to rename the command line arg to `--unstable`. I left `TestUnstable` alone because there's already a `Feature::Unstable` member, corresponding to the feature flag in the `uefi` package (as opposed to the `uefi-test-runner` package).